src/crafting.cpp

```diff
@@ -863,8 +863,19 @@ const recipe *select_crafting_recipe( int &batch_size )
             redraw = true;
             keepline = true;
         } else if (action == "FILTER") {
+          std::string desc = std::string("Special prefixes:\n") +
```

Comment:
C++ combines adjacent C-String literals into one C-String literal:

```C++
std::string text = "AB" "CD" "EF"
"GH"
"IJ";
// text contains now "ABCDEFGHIJ"
```

For translation: add `_(` and `)` around the C-String literal:

```C++
std::string text = _("this will be translated");
```

The `_` marks the literal as to be translated (there is a script that extracts all marked strings, they can be translated separately). It also calls the function to get the translation at runtime.

However, this `_(desc.c_str())` won't work because there is no string to extract here.